### ISSUE TITLE
Increase GO TLSHandshakeTimeout value

### ISSUE THREAD

**Issue by gaetanquentin** [Oxalide/vsphere-influxdb-go#68]
Since a few days we have this error:

"Could not initialize connection to vcenter cpl0115hm.jmsp.prod Post https://cpl0115hm.jmsp.prod/sdk: net/http: TLS handshake timeout"

Because the TLS  connection has become slower.

The TLSHandshakeTimeout is set at 10 secondes in GO.

Could you allow to increase it in a parameter?

Regards.

**Comment by sofixa** [comment]
Hi, 

Has something changed in your environement? 10 seconds seems a lot, does basic connectivity (ping, telnet 443, curl , etc.) towards the vCenter work ?

Cheers,
Adrian

**Comment by gaetanquentin** [comment]
yes something must have changed in our environment  but i don't know what . 

yes telnet and 'openssl s_client connect 'work fine but are very slow
